index.bs

```diff
@@ -686,40 +681,40 @@ occurred on the [=remote end=].
 
 A [=BiDi session=] has a <dfn export for=event>global event set</dfn>
 which is a [=/set=] containing the event names for events that are enabled for all
-browsing contexts. This initially contains the [=event name=] for events that
+navigables. This initially contains the [=event name=] for events that
 are <dfn export for=event>in the default event set</dfn>.
 
-A [=BiDi session=] has a <dfn export for=event>browsing context event
-map</dfn>, which is a [=/map=] with [=/top-level browsing context=] keys and values
+A [=BiDi session=] has a <dfn export for=event>navigable event map</dfn>,
+which is a [=/map=] with [=navigable/top-level traversable=] keys and values
```

Comment:
It looks like the link is broken:
```suggestion
which is a [=/map=] with [=navigable/top-level-traversable=] keys and values
```

Reply:
this line works for me and links to https://html.spec.whatwg.org/multipage/document-sequences.html#nav-top

Reply:
But it should be https://html.spec.whatwg.org/multipage/document-sequences.html#top-level-traversable

Reply:
Changed to `[=/top-level traversable=]` when references without a relation to a navigable